src/app/app.component.ts

```diff
@@ -50,6 +50,11 @@ export class AppComponent implements OnInit {
   private Config = this.config.getConfig();
   public title = this.Config.title || "Rancher's Hobby Farm";
   private logo = this.Config.logo || '/assets/default/logo.svg';
+  public aboutTitle = this.Config.about?.title || 'About HobbyFarm';
+  public aboutBody =
+    this.Config.about?.body ||
+    'HobbyFarm was lovingly crafted by @ebauman and @Oats87 from @RancherLabs';
```

Comment:
Please change this to

"HobbyFarm is lovingly crafted by the HobbyFarm team at https://github.com/hobbyfarm/hobbyfarm"